_layouts/default.html

```diff
@@ -21,7 +21,9 @@ <h1><a href="{{ "/" | absolute_url }}">{{ site.title | default: site.github.repo
           <img src="{{site.logo | relative_url}}" alt="Logo" />
         {% endif %}
 
+        {% if site.hide_description_in_side_bar != true %}
```

Comment:
I'd opt with to avoid to overcharge configuration with a merelly boolean applied over `site.description`

```suggestion
        {% if site.description != false %}
```

Reply:
Resolved at [`d3fd543` (#132)](https://github.com/pages-themes/minimal/pull/132/commits/d3fd543dd08c5ed2a83287e85b2fc3c17f10abb1)